### ISSUE TITLE
[BUG]: Displayed grid window is too small

### ISSUE THREAD

**Issue by 3dprinting-source** [jneilliii/OctoPrint-BedLevelVisualizer#638]
For the ENder 3 S1 Pro

![Capture200](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/assets/74247897/be59fd22-ba38-428f-8b17-c4ef1f014a35)

**Describe the bug**

**Expected behavior**

**Debug Log**

**Screenshots**

**Firmware and Version**

**Comment by jneilliii** [comment]
Yeah, the layout was optimized for a 10x10 grid, which barely fit on the stock UI. You might be able to use UI Customizer plugin to make your layout like this and be able to see more of the data.

![image](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/assets/5249455/2af47386-9c5b-44ee-b233-d8d92c48f060)

![image](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/assets/5249455/ec91e2be-1040-49cd-aaba-858ebe03d2ae)

Give the tab a lot more width to work with...

![image](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/assets/5249455/c6640264-ab6d-46a3-be97-8b9ba5b89051)
